src/components/application_manager/test/commands/mobile/on_system_request_notification_test.cc

```diff
@@ -119,6 +177,9 @@ TEST_F(OnSystemRequestNotificationTest, Run_ProprietaryType_SUCCESS) {
             (*msg)[strings::params][strings::protocol_type].asInt());
   ASSERT_EQ(CommandImpl::protocol_version_,
             (*msg)[strings::params][strings::protocol_version].asInt());
+#ifdef EXTENDED_POLICY
```

Comment:
@LevchenkoS It seems that `EXPECT_EQ` is more appropriate here as failure is not going to cause further problems.